developers.adoc

```diff
@@ -52,6 +52,20 @@ several ways:
 ** not packaged, used as a source JS file on a website
 ** packaged as https://www.npmjs.com/[npm] package
 
+=== Development loop
+
+The general process of making and testing a change once you have the
+prerequistes and sources ready:
+
+* Making changes in the `compiler` repo:
+  * Perform a "stage build" with SBT: `sbt compilerJVM/stage`
+  * Use your new compiler directly: `./jvm/target/universal/stage/bin/kaitai-struct-compiler <args>`
```

Comment:
Tip: I have the `./jvm/target/universal/stage/bin` directory permanently in my `PATH`, which is super useful because I never have to type that long path myself and can invoke `kaitai-struct-compiler` from any working directory.

And when you use terminal autocompletion to "help you" with typing this long path, you can accidentally call the **wrong** script in <code>jvm/target/universal/<ins><strong>scripts</strong></ins>/bin/kaitai-struct-compiler</code>, which won't work and will give you `Error: Could not find or load main class io.kaitai.struct.JavaMain`, see https://github.com/kaitai-io/kaitai_struct/issues/1110. So it's error prone, which is why I wouldn't recommend it.